### PR TITLE
Custom filter title

### PR DESCRIPTION
**Pull request by listepo**
for example:  
```javascript
  customFilterTitle(filterTitle) {
    return `filter[${filterTitle}]`;
  }
```

### CODE REVIEW

File: addon/components/models-table-server-paginated.js [hunk 1]
```diff
@@ -195,6 +197,15 @@ export default ModelsTable.extend({
     return query;
   },
 
+  /**
+   * Set custom filter title
+   *
+   * @type {string} filter title
+   */
+  customFilterTitle(filterTitle) {
+    return filterTitle;
```
Comment (onechiporenko) [inline addon/components/models-table-server-paginated.js]: I think, it's better to do:
```javascript
/**
 * @param {object} column
 * @return {string}
 */
getCustomFilterTitle(column) {
  get(column, 'filteredBy') || get(column, 'propertyName')
}
```

Comment (listepo) [inline addon/components/models-table-server-paginated.js]: @onechiporenko, I agree

**Comment by listepo** [comment]
ping

**Comment by onechiporenko** [comment]
Thanks, @listepo

**Pull request merged**